lib/API/JSON/v1/Subscribers.php

```diff
@@ -385,7 +385,6 @@ public function save($data = []) {
         isset($data['status'])
         && ($data['status'] === SubscriberEntity::STATUS_UNSUBSCRIBED)
         && ($oldSubscriber instanceof Subscriber)
-        && ($oldSubscriber->status === SubscriberEntity::STATUS_SUBSCRIBED)
```

Comment:
@pavel-mailpoet  This should probably be `&& ($oldSubscriber->status !== SubscriberEntity::STATUS_UNSUBSCRIBED)`, otherwise it will write to unsubscribe log every time an unsubscribed user is edited.

Reply:
@wxa great catch. That would be a big problem.